Error: Can't resolve 'concat-style-loader'

### Problem

Running any of the build commands using the current version of starter-theme results in an error while trying to process `styles/theme.scss.liquid` using the new `concat-style-loader` package.

`Module not found: Error: Can't resolve 'concat-style-loader'`

![image](https://user-images.githubusercontent.com/454095/38900065-f5d93238-424e-11e8-908d-bc103a6ea1a3.png)

### More Information

Commenting out the *.scss.liquid import from `assets/theme.js` does allow the theme to build, however it seems like PR #488 is missing the inclusion of the new `concat-style-loader` in the package files.

--- a/packages/slate-tools/package.json
+++ b/packages/slate-tools/package.json
@@ -30,6 +30,7 @@
     "babel-loader": "7.1.4",
     "chalk": "2.3.2",
     "clean-webpack-plugin": "0.1.19",
+    "concat-style-loader": "^1.0.0-alpha.27",
     "console-control-strings": "^1.1.0",
     "copy-webpack-plugin": "^4.2.3",
     "cors": "^2.8.4",

